src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -351,6 +367,416 @@ def output_gds(self, value: str) -> None:
         self.attr_setter("_output_gds", value)
 
 
+    @property
+    def output_netlist(self) -> str:
+        """
+        Get the path to the output netlist file.
+
+        :return: The path to the output netlist file.
+        """
+        try:
+            return self.attr_getter("_output_netlist", None)
+        except AttributeError:
+            raise ValueError("Nothing set for the path to the output netlist file yet")
+
+    @output_netlist.setter
+    def output_netlist(self, value: str) -> None:
+        """Set the path to the output netlist file."""
+        if not (isinstance(value, str)):
+            raise TypeError("output_netlist must be a str")
+        self.attr_setter("_output_netlist", value)
+
+
+    @property
+    def power_nets(self) -> List[str]:
+        """
+        Get the list of power.
+
+        :return: The list of power.
```

Comment:
Seems like the regeneration didn't go through?

Reply:
Some weird merge artifacts here, I think. This just needs to be deleted.

Reply:
Nope, I literally just typo'd in `generate_properties.py`

Reply:
Fixed